<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSmartCode</class>
 <widget class="QMainWindow" name="SSmartCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSmartCode</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="layTopSide">
        <item>
         <widget class="QSplitter" name="splMain">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="wLeftSide" native="true">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <widget class="QWidget" name="pLeftSideTree">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>1</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,10">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="bAddFile">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>+</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bDeleteEntry">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>-</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bStart">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>|&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTreeWidget" name="tvProjectFiles">
                  <property name="styleSheet">
                   <string notr="true">background: rgba(255,255,255,0.0); border: 0px solid black;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="pLeftSideList">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListWidget" name="lwProjectFiles"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="wCentralSide" native="true">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="swCentral">
              <widget class="QWidget" name="pMainSideCE">
               <layout class="QVBoxLayout" name="codeEditorLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2"/>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="wRightSide" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QStackedWidget" name="swRight">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="pOutputView">
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPlainTextEdit" name="pteResult"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="pFindReplace">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>Найти-Заменить</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                     <string>Режим</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_9">
                     <property name="spacing">
                      <number>1</number>
                     </property>
                     <property name="leftMargin">
                      <number>3</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>3</number>
                     </property>
                     <property name="bottomMargin">
                      <number>9</number>
                     </property>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_6">
                       <item>
                        <widget class="QRadioButton" name="rbRegEx">
                         <property name="text">
                          <string>Регулярное</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QRadioButton" name="radioButton_2">
                         <property name="text">
                          <string>Текст</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <widget class="QRadioButton" name="radioButton_3">
                         <property name="text">
                          <string>Маска</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QCheckBox" name="checkBox">
                         <property name="text">
                          <string>Регистр</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLineEdit" name="lineEdit">
                      <property name="placeholderText">
                       <string>Выражение поиска</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <widget class="QPushButton" name="pushButton_2">
                      <property name="text">
                       <string>Найти</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton">
                      <property name="text">
                       <string>Найти все</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QLineEdit" name="lineEdit_2">
                      <property name="placeholderText">
                       <string>Выражение замены</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QPushButton" name="pushButton_4">
                      <property name="text">
                       <string>Заменить</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_3">
                      <property name="text">
                       <string>Заменить все</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wBotomSide" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>250</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="aCreateProject"/>
    <addaction name="aOpenProject"/>
    <addaction name="aSaveProject"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="aFindReplace"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Панели</string>
    </property>
    <addaction name="aPanelFindReplace"/>
    <addaction name="aProjectTree"/>
    <addaction name="action_9"/>
    <addaction name="aTerminal"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aCreateProject">
   <property name="text">
    <string>Создать проект</string>
   </property>
  </action>
  <action name="aOpenProject">
   <property name="text">
    <string>Открыть проект</string>
   </property>
  </action>
  <action name="aSaveFile">
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Создать файл</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить проект</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Закрыть проект</string>
   </property>
  </action>
  <action name="aFindReplace">
   <property name="text">
    <string>Найти-Заменить</string>
   </property>
  </action>
  <action name="aTerminal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Терминал</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Вывод</string>
   </property>
  </action>
  <action name="aPanelFindReplace">
   <property name="text">
    <string>Найти-Заменить</string>
   </property>
  </action>
  <action name="aProjectTree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Проекты</string>
   </property>
  </action>
  <action name="aSaveProject">
   <property name="text">
    <string>Сохранить проект</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
